mava/environment_loop.py

```diff
@@ -481,10 +494,46 @@ def should_terminate(episode_count: int, step_count: int) -> bool:
                 num_steps is not None and step_count >= num_steps
             )
 
+        def should_run_loop(eval_condtion: Tuple) -> bool:
```

Comment:
Why do we define this function inside the run function and not inside the ParallelEnvironmentLoop class?

Reply:
And also the should_terminate function. Maybe there is a reason for these functions being here?

Reply:
So this is a side-effect of our environment loop being based on [acme's one ](https://github.com/deepmind/acme/blob/881f1b01c01a73e0a0df9bd0ac5c205028ca1b31/acme/environment_loop.py#L152). 

From a function level, the only real difference I see is you don't need to call `self.[function]`. Also might be slightly faster being called this way, although I am not sure about that.

Reply:
I see thanks. We can keep it as is :)